README.md

```diff
@@ -425,11 +428,19 @@ Write the `contents` to `filepath` at the given random access position. When `po
 
 Moves the file located at `filepath` to `destPath`. This is more performant than reading and then re-writing the file data because the move is done natively and the data doesn't have to be copied or cross the bridge.
 
+Note: Overwrites existing file in Windows.
+
+### `copyFolder(srcFolderPath: string, destFolderPath: string): Promise<void>`
+
+Copies the contents located at `srcFolderPath` to `destFolderPath`.
+
+Note: Windows only. This method is recommended when directories need to be copied from one place to another.
+
 ### `copyFile(filepath: string, destPath: string): Promise<void>`
 
 Copies the file located at `filepath` to `destPath`.
 
-Note: On Android copyFile will overwrite `destPath` if it already exists. On iOS an error will be thrown if the file already exists.
+Note: On Android and Windows copyFile will overwrite `destPath` if it already exists. On iOS an error will be thrown if the file already exists.
```

Comment:
it would be good/important to match API behavior across platforms but I realize that's a bigger issue than this PR

Reply:
I agree, but it is outside of the scope of this PR for the time being.